package.json

```diff
@@ -1,34 +1,34 @@
 {
-    "name": "puma",
-    "version": "0.0.1",
-    "description": "Easy Metaprograming for JS",
-    "repository": "http://github.com/emravera/puma",
-    "author": {
-        "name": "UTN-LIS",
-        "email": "alexis.ferreyra@gmail.com",
-        "url": "http://github.com/emravera/puma"
-    },
-    "keywords": [
-        "puma"
-    ],
-    "main": "dist/pumascript.js",
-    "engines": {
-        "node": ">= 0.8.0"
-    },
-    "license": "UTN-LIS",
-    "devDependencies": {
-        "grunt-contrib-clean": "~0.7.0",
-        "grunt-contrib-jshint": "~0.11.3",
-        "grunt-contrib-qunit": "~0.5.1",
-        "grunt-contrib-requirejs": "0.4.4",
-        "grunt": "~0.4.5",
-        "jshint-stylish": "~1.0.1",
-        "load-grunt-tasks": "~3.1.0"
```

Comment:
This dev dependency is esential for the grunt task to run. It is missing from the package.json and for that reason all the travis CI jobs are failing.